### ISSUE TITLE
On the fly configuration

### ISSUE THREAD

**Issue by thelostt** [muxx-utils/muxx#1]
## TL;DR

`muxx-send(1)` tool for altering the window manager configurations on the fly, using [xcb_client_message_event_t](ftp://www.x.org/pub/X11R7.7/doc/man/man3/xcb_client_message_event_t.3.xhtml), and a minimal configuration file, both generated as you change the values on the fly, or manually updating configs.
## Implementation

`muxx-send -m <message> -- [args]...` is the low-level tool for sending client message events to other tools. Specialize this tool, so that it has a nicer interface.

Goals are:
- Movement and resizing with less typing;
- Configuration management;
- Modularizing functionality;
